Dockerfile

```diff
@@ -1,13 +1,32 @@
 FROM python:3.6
 
-# prepare directory
+# explicitly set user/group IDs
+RUN groupadd -r -g 999 kqueen && \
+    useradd -r -g kqueen -u 999 -m -s /sbin/nologin -c "Kqueen user" kqueen
+
+COPY . /code
+
 WORKDIR /code
 
-# copy app
-COPY . .
+RUN mkdir -p log/kqueen
```

Comment:
You should expect container to be running on read-only mode so please move logs out of source code.

Reply:
imo there are 3 ways to  provide logging directory:
1) in docker file
2) in entry point
3) in python backend
which one do you prefer? or you have any another proposals?

Reply:
If fine with providing loggin directory in this place but please don't log into same directory as source code is living.